packages/maskbook/src/plugins/Voicechat/UI/VoicechatInlay.tsx

```diff
@@ -0,0 +1,254 @@
+                        <Box mr={0.5} style={{ marginTop: '6px' }}>
+                            <SpeakIcon />
+                        </Box>
+                        <Typography component="p" sx={{ fontWeight: isSpeaking ? 'bold' : 'regular' }}>
+                            {peerMedia.isLocal ? t('plugin_voicechat_you') : peerMedia.username}
+                        </Typography>
+                    </Box>
+                )
+            })}
+        </div>
+    )
+}
+
+function JoinButton(props: JoinButtonProps) {
+    const { t } = useI18N()
+
+    return props.localPeerMediaElement ? null : (
+        <Button
+            color="primary"
+            variant="contained"
+            style={{ color: '#fff' }}
+            onClick={() => props.joinChat()}
+            startIcon={props.isJoining ? <CircularProgress style={{ color: 'white' }} size={24} /> : null}>
+            {t('plugin_voicechat_join_conversation')}
+        </Button>
+    )
+}
+
+function MediaSelection(props: MediaSelectionProps) {
+    return (
+        <FormControl>
+            <Select
+                value={props.activeMediaDevice}
+                MenuProps={{ container: PortalShadowRoot }}
+                onChange={(e) => {
+                    if (props.localPeerMediaElement) {
+                        changeMediaDevice(e.target.value, props.localPeerMediaElement.username)
+                    }
+                }}>
+                {props.mediaDevices.map((device) => (
+                    <MenuItem key={device.deviceId} value={device.deviceId}>
+                        {device.label}
+                    </MenuItem>
+                ))}
+            </Select>
+        </FormControl>
+    )
+}
+
+function MuteSwitch(props: MuteSwitchProps) {
+    return (
+        <Button
+            onClick={() => {
+                props.setMuted(!props.isMuted)
+                props.setIsMuted(!props.isMuted)
+            }}
+            variant="outlined"
+            style={{ minHeight: '33px', minWidth: '20px' }}>
+            {props.isMuted ? <VolumeOffOutlined fontSize={'small'} /> : <VolumeUpOutlined fontSize={'small'} />}
+        </Button>
+    )
+}
+
+function DisconnectButton(props: DisconnectButtonProps) {
+    return (
+        <Button
+            onClick={() => {
+                props.disconnectVoice()
+                props.setIsMuted(false)
+            }}
+            variant="outlined"
+            style={{ minHeight: '33px', minWidth: '20px', marginLeft: '5px' }}
+            color="secondary">
+            <CloseOutlined fontSize={'small'} />
+        </Button>
+    )
+}
+
+function Settings(props: SettingsProps) {
+    const {
+        mediaDevices,
+        activeMediaDevice,
+        localPeerMediaElement,
+        isMuted,
+        setIsMuted,
+        setMuted,
+        disconnectVoice,
+    } = props
+
+    return localPeerMediaElement ? (
+        <Box
+            px={2}
+            py={1}
```

Comment:
hi, please do not use abbreviations. Use their full versions like "paddingY" or something.